CHANGELOG.md

```diff
@@ -12,7 +12,8 @@ If you're using Nunjucks, you can now add classes to the character count compone
 
 ### Fixes
 
-- [Pull request #1690: Don't unneccesarily self-close tags](https://github.com/alphagov/govuk-frontend/pull/1690)
+- [Pull request #1698: Fix documentation so that `id` option in date input is required](https://github.com/alphagov/govuk-frontend/pull/1698)
```

Comment:
Trying to work out if this is useful to add to the release notes for Frontend – it feels like it might be an internal thing, as the value is ultimately 'delivered' in the Design System? Thoughts? /cc @m-green

Reply:
It seems best to leave it out of the release note - because we're not changing how Frontend behaves, we're just updating the documentation to match the reality.

Reply:
Yeah we don't update the CHANGELOG for documentation, my mistake